frontends/web/src/api/config.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { apiGet, apiPost } from '../utils/request';
+
+export type TFavorites = {
+    readonly [key in string]: boolean;
+}
+
+export function getAccountFavorites() {
+    return apiGet('config')
+        .then(({ frontend = {} }): Promise<TFavorites> => {
+            return frontend.accountFavorites || {};
+        });
+}
+
+export function setAccountFavorite(
+    accountId: string,
+    favorite: boolean,
+) {
+    return apiGet('config').then(config => {
+        const { frontend = {} } = config;
+        return apiPost('config', {
```

Comment:
use `setConfig()` from utils/config.js, which already takes care to only update the diff. It is also used in settings.tsx.

Reply:
PTAL